pkg/doc/verifiable/credential.go

```diff
@@ -319,6 +359,22 @@ func WithPublicKeyFetcher(fetcher PublicKeyFetcher) CredentialOpt {
 	}
 }
 
+// WithJSONSchemaCacheParam option is used to apply JSON Schema caching.
+func WithJSONSchemaCacheParam(param *JSONSchemaCacheParam) CredentialOpt {
```

Comment:
The option should accept a constructed cache so that caches can be shared.

Reply:
It could also make sense to also have the schema loader as well so that there are no globals. In this case, The object passed-in would be a wrapper of cache and schema loader.

Reply:
Makes sense @troyronda , I missed this point, shame on me.
I pushed a new version, no globals, the cache is configurable.